cfpb/consumerfinance.gov
# update CCDB5@2.4.10

---




## Changes
- Updating CCDB5 to latest to include bug fixes.
- Prune incomplete date intervals from charts.
- Updated CCDB Cypress test.

## How to test this PR

1. Pull down
2. run ./frontend.sh
3. visit http://localhost:8000/data-research/consumer-complaints/search/
4. Mouse over chart and confirm you don't see any incomplete start or end intervals

## Screenshots
<img width="1324" alt="Screen Shot 2022-03-02 at 10 30 34 PM" src="https://user-images.githubusercontent.com/1038306/156648168-2fa66af7-886f-4165-ae25-6c9c99357c47.png">
